### ISSUE TITLE
Add support for chord symbol description in Chord Symbol component

### ISSUE THREAD

**Issue by amypellegrini** [amypellegrini/react-scorewriter#13]
Example:

Cmaj7 chord symbol should have an `aria-details` attribute with value "C major seventh chord".
